### PR TITLE
Create xdg-desktop-portal-hyprland

### PR DESCRIPTION
**Pull request by odomingao**

### CODE REVIEW

File: apparmor.d/groups/freedesktop/xdg-desktop-portal-hyprland [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = @{lib}/xdg-desktop-portal-hyprland
+profile xdg-desktop-portal-hyprland @{exec_path} {
+  include <abstractions/base>
+  include <abstractions/audio>
+  include <abstractions/audio-client>
```
Comment (roddhjav) [inline apparmor.d/groups/freedesktop/xdg-desktop-portal-hyprland:13]: Only use `audio-client`, not both.

File: apparmor.d/groups/freedesktop/xdg-desktop-portal-hyprland [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  include <abstractions/dconf-write>
+  include <abstractions/desktop>
+  include <abstractions/dri-enumerate>
```
Comment (roddhjav) [inline apparmor.d/groups/freedesktop/xdg-desktop-portal-hyprland:16]: Remove it, it is included in `graphics`

**Comment by odomingao** [comment]
fixed

**Comment by roddhjav** [comment]
Thanks, merged.

**Pull request merged**